src/config/main-config.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
+const path = require('path');
+const logger = require('morgan');
+const cors = require('cors')
+const bodyParser = require('body-parser');
+
+module.exports = {
+  init(app, express) {
```

Comment:
You can just import `express` in this file, rather than passing it as a parameter

Reply:
Removed parameter and added dependency.